gcp/api/integration_tests.py

```diff
@@ -354,6 +354,62 @@ def test_query_batch(self):
             ]
         }, response.json())
 
+  def test_query_package(self):
+    """Test query by package."""
+    response = requests.post(
+        _api() + '/v1/query',
+        data=json.dumps({
+            'package': {
+                'ecosystem': 'Maven',
+                'name': 'org.apache.tomcat:tomcat',
+            }
+        }))
+
+    result = response.json()
+    vulns_first = set(v['id'] for v in result['vulns'])
+    self.assertIn('next_page_token', result)
+
+    response = requests.post(
+        _api() + '/v1/query',
+        data=json.dumps({
+            'package': {
+                'ecosystem': 'Maven',
+                'name': 'org.apache.tomcat:tomcat',
+            },
+            'page_token': result['next_page_token'],
+        }))
+
+    result = response.json()
+    vulns_second = set(v['id'] for v in result['vulns'])
+
+    self.assertEqual(set(), vulns_first & vulns_second)
```

Comment:
Could we use `vulns_first.intersection(vulns_second)` here instead of `&` for greater readability? See also the discussion at https://docs.python.org/3/library/stdtypes.html#frozenset.intersection

Reply:
good call, done!